Configuration/SecurityController.m

```diff
@@ -7,7 +7,4 @@ +(bool)rootDetection{
 +(bool)debugDetection{
   return NO;
 }
-+(bool)debugLogs{
-  return YES;
```

Comment:
Isn't this filem modified by the configuration script? Was this method removed by it?

Reply:
it's, the Configurator v5 did remove this